docs/LoaderLayerInterface.md

```diff
@@ -1379,6 +1379,11 @@ layer is not applied.
 So if an override meta layer wants to mix default and custom layer locations,
 the override paths must contain both custom and default layer locations.
 
+* If both the override meta layer is present and contains `override_paths`, the
```

Comment:
"If both the override meta layer is present and contains" would be clearer as "If the override layer is both present and contains".